vaadin-chip-combobox/src/main/java/software/xdev/vaadin/chips/ChipComponent.java

```diff
@@ -64,6 +66,8 @@ protected void initUI()
 		this.btnDelete.setSizeUndefined();
 		
 		this.getContent().setSpacing(false);
+		this.getContent().setAlignItems(Alignment.CENTER);
+		this.getContent().setJustifyContentMode(JustifyContentMode.CENTER);
```

Comment:
I think this also works without ``JustifyContent``.
Any explicit reasons why this was added?

Reply:
I've added it mainly due to the request in the original issue, because it'll allow potential width adjustments while keeping the label centered.

Reply:
Ah okay, I see.

> // next line is optional, but would be nice if a feature is added to stretch all chips to the same width

However I don't think as long as there is a feature for that we shouldn't bloat the code with unnecessary changes.
Please remove it anyway if there are no any other arguments.

Reply:
Removed the line in the latest commit